main/main.cpp

```diff
@@ -337,7 +337,7 @@ Error Main::setup(const char *execpath, int argc, char *argv[], bool p_second_ph
 
 	String video_driver = "";
 	String audio_driver = "";
-	String game_path = ".";
+	String game_path;
```

Comment:
Not sure how, but I have a hint that this might have broken the `--doctool` argument parsing. It used to be `godot --doctool .`, but now you need to pass `godot --doctool . .` to get it to work.